### PR TITLE
(Debugging - WIP) Add feature to restart active challenge workers every 3 days.

### PR DESCRIPTION
**Pull request by KhalidRmb**
- Included a cron job which runs everyday, to filter challenges which were last restarted 3 days ago.
- Restart the workers for the above challenges.

**TODO (In another PR):**
- Include a new field in the Challenge model to represent a defective challenge as an additional filter for this corn job. (Instead of restarting every challenge periodically.)

### CODE REVIEW

File: apps/challenges/migrations/0065_added_last_started_at_field_to_challenges.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('challenges', '0064_add_is_restricted_to_select_one_submission_field_in_phase_model'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='challenge',
+            name='last_started_at',
```
Comment (deshraj) [inline apps/challenges/migrations/0065_added_last_started_at_field_to_challenges.py:17]: Can we please change this field to `worker_last_started_at` to be more clear ?

File: apps/challenges/models.py [hunk 2]
```diff
@@ -164,6 +167,11 @@ def is_active(self):
             return True
         return False
 
+    @property
+    def three_days_since_last_start(self):
```
Comment (deshraj) [inline apps/challenges/models.py:171]: nit: can we please change the name to `n_days_since_last_worker_start`

File: apps/challenges/restart_workers_cron.py [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+    count, failures = response["count"], response["failures"]
+
+    if(count == challenges.count()):
+        logger.info("All challenge workers restarted succesfully.")
+    else:
+        for fail in failures:
+            logger.warning("Worker restart for challenge {} failed. Error: {}".format(fail["challenge_pk"], fail["message"]))
+
+
+'''
```
Comment (deshraj) [inline apps/challenges/restart_workers_cron.py:21]: Lets remove this commented part if we are not using it.

File: apps/challenges/aws_utils.py [hunk 4]
```diff
@@ -1,3 +1,4 @@
+from datetime import date
```
Comment (Ram81) [inline apps/challenges/aws_utils.py:1]: @KhalidRmb can we put all the `from` imports after `import` statements

**Pull request closed without merging**